packages/xrpl/integrationTestSequencer.js

```diff
@@ -0,0 +1,33 @@
+const Sequencer = require('@jest/test-sequencer').default
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
+    // Test structure information
+    // https://github.com/facebook/jest/blob/6b8b1404a1d9254e7d5d90a8934087a9c9899dab/packages/jest-runner/src/types.ts#L17-L21
+    const copyTests = Array.from(tests)
+    const sorted = copyTests.sort((testA, testB) => {
+      // We run escrow tests first because they rely on the ledger number via ledger_accept
+      if (testA.path.includes('escrow') && testA.path.includes('escrow')) {
+        return -1
+      }
+
+      if (testB.path.includes('escrow')) {
+        return 1
+      }
+
+      if (testA.path.includes('finalTest')) {
+        return 1
+      }
+
+      if (testB.path.includes('finalTest')) {
+        return -1
+      }
+
+      return testA.path > testB.path ? 1 : -1
+    })
+    console.log('sorted', sorted)
```

Comment:
You should remove this console.log.  It makes the test output very noisy.

Reply:
Yes, it was there only for debugging